app/views/admin/enterprises/_change_type_form.html.haml

```diff
@@ -36,9 +33,6 @@
             .top
```

Comment:
Unless putting back `monthly-pricing-description`, the `top` CSS class might no longer be necessary here.

Reply:
you should be right, I'll have a look at this